### DISCUSSION TITLE
iFrame Embed code on external site

### DISCUSSION THREAD

**Discussion by DCurle** [jgraph/drawio-desktop#1637] [Q&A]
From draw.io, I have selected File - Embed - iFrame to get embed code to use on a sharepoint site. The code refers to src="https://viewer.diagrams.net..., which is outside my organisation. Is there any way to change it to be inside my organization? Thank you.

**Comment by davidjgraph** [comment]
You could host the drawio repo internally within your domain.

**Reply by DCurle** [reply]
Thank you for your response. 
Please say more on what this means and how it is achieved?
